repositories/local_storage.go

```diff
@@ -16,7 +17,13 @@ const filePath = "resources/pokemons.csv"
 
 func (l LocalStorage) Write(pokemons []models.Pokemon) error {
 	file, fErr := os.Create(filePath)
-	defer file.Close()
+	defer func() {
```

Comment:
As this is just an example it was not necessary to change this but nice handle of errors